### DISCUSSION TITLE
샌드박스/랩 활성화를 위한 마이크로소프트 계정 연결 문제

### DISCUSSION THREAD

**Discussion by borimong** [devrel-kr/HackaLearn#46] [2 Q&A - 클라우드 스킬 챌린지] [answered]
연습 - Azure 정적 웹앱 만들기 단계에서

샌드박스나 랩을 활성화하려면 메일이 필요합니다.
샌드박스나 랩을 활성화하려면 Microsoft 계정이 유효한 메일에 연결되어야 합니다. Microsoft 계정 설정으로 이동하여 이메일을 연결하고 다시 시도하세요.
자세한 내용은 문제 해결 지침 페이지를 확인하세요.

라는 안내가 뜨는데 어떻게 해결해야 할까요..? 
유효한 메일 등록되어 있음에도 계속해서 저런 안내창이 뜹니다ㅠㅠ

**Comment by minhoryang** [comment, marked as answer]
우선 해당 메일로 https://outlook.com/ 에 로그인이 되시는지 알려주시겠어요?

**Reply by minhoryang** [reply]
그 이후 Azure의 체험계정에 등록해주셔야할 것 같구요. (이 과정에서 학생인증이나 카드인증이 필요할 수 있습니다.)

**Reply by borimong** [reply]
로그인도 되고 azure 체험 계정 등록까지 마쳤는데 여전히 같은 문제가 발생합니다..ㅠㅠ

**Reply by justinyoo** [reply]
크롬의 incognito 모드 혹은 엣지의 InPrivate 모드로 웹 브라우저를 열고 거기서 로그인한 후 샌드박스 구독을 활성화 시켜 보세요.

**Reply by justinyoo** [reply]
그리고, 어느 부분에서 에러가 발생하는지 스크린샷 같은 것을 첨부해 주시면 알려드리기가 수월합니다.

**Reply by borimong** [reply]
문제 해결되었습니다!! 정말 감사합니다!!! :)

**Reply by minhoryang** [reply]
Answered 해주셔요 👍

**Comment by justinyoo** [comment]
https://account.microsoft.com 에서 계정을 하나 생성하셔야 합니다. 굳이 outlook, live, hotmail 계정이 아니어도 상관은 없어요. 하지만, 계정을 하나 생성하셔야 무료로 애저 샌드박스 구독을 이용하실 수 있습니다.
